modules/110-istio/openapi/config-values.yaml

```
@@ -434,44 +515,46 @@ properties:
               cpu: "55m"
               memory: "256Mi"
             limits:
-              cpu: 2
+              cpu: "2"
               memory: "2Gi"
         properties:
           mode:
             type: string
             description: |
-              The mode for managing resource requests. Classical `Static` requests/limit or `VPA`.
+              Resource management mode:
+              - `Static` is a classic one. In it, you specify requests/limits. The parameters of this mode are defined in the [static](#parameters-resourcesmanagement-static) parameter section;
```

```suggestion
              - Allows you to specify requests/limits. The parameters of this mode are defined in the [static](#parameters-resourcesmanagement-static) parameter section;
```
People could think the "classic one" means "defafult".